services/notifications/README.md

```diff
@@ -35,6 +43,7 @@ templates
 ```
 
 Custom email templates referenced via `NOTIFICATIONS_EMAIL_TEMPLATE_PATH` must also be located in subfolders `shares` and `spaces` and must have the same names as the embedded templates. It is important that the names of these files and  folders match the embedded ones.
+In the subfolder `html` contains a default HTML template provided by ocis. The images can be embedded in a custom HTML template as a CID source ```<img src="cid:logo-mail.gif" alt="logo-mail"/>``` The image files should be located in `html/img` subfolder. Supported image types are png, jpeg, and gif.
```

Comment:
```suggestion
The `html` subfolder contains a default HTML template provided by ocis. When using a custom HTML template, images can either be linked with standard HTML code or also be embedded as a CID source ```<img src="cid:logo-mail.gif" alt="logo-mail"/>``` In this case, image files must be located in the `html/img` subfolder. Supported image types are png, jpeg, and gif.
```
What about svg images, are we open for this when embedding?
Note that I have read (and I may be wrong) that embedding images via CID can cause [issues_1](https://sendgrid.com/blog/embedding-images-emails-facts/) or [issues_2](https://mailtrap.io/blog/embedding-images-in-html-email-have-the-rules-changed/) in web clients. Should we drop a note here? like:
`Consider that embedding images via a CID resource may not be fully supported in all email web clients.`

Reply:
@mmattel Good points
1. We do not support svg when embedding. 
2. Let's make a notice about embedding and add that hosted images source are also available like `<img src="https://raw.githubusercontent.com/owncloud/core/master/core/img/logo-mail.gif" alt="logo-mail"/>` 
Or better make the hosted images as a default way?
BTW How it works in ownCloud 10?
FYI @kobergj @micbar

Reply:
> Or better make the hosted images as a default way?

CID linking is ok then admins can see how that has to look like.
Embedding via standard html is imho commonly known.